src/tests/ftest/util/ior_test_base.py

```diff
@@ -98,12 +95,10 @@ def get_job_manager_command(self):
 
         """
         # Initialize MpioUtils if IOR is running in MPIIO mode
-        if self.ior_cmd.api.value == "MPIIO":
```

Comment:
Hrm.  I thought my patch included a fix for this also.  I prefer to keep the test of `self.ior_cmd.api.value` here but have it test for `"MPIIO"` or `"DAOS'` and `self.fail()` if it's neither.  That future proofs against any new APIs that need to be dealt with here rather than somebody have to debug it to figure out this code needs to handle new APIs.

Reply:
Done

Reply:
Done but not pushed yet?

Reply:
It has been pushed. this is how it looks like now.
if self.ior_cmd.api.value in ["MPIIO", "DAOS"]:
  mpio_util = MpioUtils()
  if mpio_util.mpich_installed(self.hostlist_clients) is False:
    self.fail("Exiting Test: Mpich not installed")
else:
  self.fail("Unsupported IOR API")
return os.path.join(mpio_util.mpichinstall, "bin", "mpirun")